workbench-core/environments/src/services/environmentService.test.ts

```diff
@@ -1042,5 +1042,104 @@ describe('EnvironmentService', () => {
         error: undefined
       });
     });
+    test('failed because ETC does not exist', async () => {
+      // BUILD
+      // Get env metadata
+      const batchItems = {
+        Responses: {
+          [TABLE_NAME]: [
+            marshall({ ...projItem, resourceType: 'project' }),
+            marshall({ ...datasetItem, resourceType: 'dataset' })
+          ] // Order is important
+        }
+      };
+      // @ts-ignore
```

Comment:
What is getting thrown here?

Reply:
Looks like this was not needed. Removing